src/shipit_dashboard/requirements.nix

```diff
@@ -35,7 +35,9 @@ let
           for dep in ${builtins.concatStringsSep " " (builtins.attrValues pkgs)}; do
             if [ -d "$dep/bin" ]; then
               for prog in "$dep/bin/"*; do
-                ln -s $prog $out/bin/`basename $prog`
+                if [ -f $prog ]; then
```

Comment:
we shouldn't edit ``requirements.nix`` but fix upstream in ``pypi2nix`` and then regenerate this file.